src/components/organisms/OrganismArticle.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="mdc-card">
+    <div class="header">
+      <h1 class="mdc-typography--headline5 no-margin">{{ storage }}</h1>
```

Comment:
Remove this class as no specific styles are applied.
```suggestion
      <h1 class="mdc-typography--headline5">{{ storage }}</h1>
```